src/libstd/sys/common/unwind/mod.rs

```diff
@@ -211,6 +215,10 @@ pub fn begin_unwind_fmt(msg: fmt::Arguments, file_line: &(&'static str, u32)) ->
 }
 
 /// This is the entry point of unwinding for panic!() and assert!().
+#[doc(hidden)]
+#[unstable(feature = "libstd_sys_internals",
+           reason = "this function is considered exported for some reason",
```

Comment:
I think that these reasons may want to be tweaked a bit here and there, "for some reason" in this case is because it's actually exported from the standard library under `std::rt`. It is never intended to be stabilized, however, it's just an internal implementation detail of the `panic!` macro